[2025-07-09T19:11:59.119+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-07-09T19:11:59.156+0000] {taskinstance.py:2631} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: process_exercises.process_exercises manual__2025-07-09T19:11:55.126986+00:00 [queued]>
[2025-07-09T19:11:59.167+0000] {taskinstance.py:2631} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: process_exercises.process_exercises manual__2025-07-09T19:11:55.126986+00:00 [queued]>
[2025-07-09T19:11:59.168+0000] {taskinstance.py:2884} INFO - Starting attempt 1 of 1
[2025-07-09T19:11:59.210+0000] {taskinstance.py:2907} INFO - Executing <Task(SparkSubmitOperator): process_exercises> on 2025-07-09 19:11:55.126986+00:00
[2025-07-09T19:11:59.218+0000] {logging_mixin.py:190} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=5346) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-07-09T19:11:59.217+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'process_exercises', 'process_exercises', 'manual__2025-07-09T19:11:55.126986+00:00', '--job-id', '250', '--raw', '--subdir', 'DAGS_FOLDER/process_exercises.py', '--cfg-path', '/tmp/tmp2vdendws']
[2025-07-09T19:11:59.220+0000] {standard_task_runner.py:72} INFO - Started process 5347 to run task
[2025-07-09T19:11:59.220+0000] {standard_task_runner.py:105} INFO - Job 250: Subtask process_exercises
[2025-07-09T19:11:59.341+0000] {task_command.py:467} INFO - Running <TaskInstance: process_exercises.process_exercises manual__2025-07-09T19:11:55.126986+00:00 [running]> on host 38b7408c68d7
[2025-07-09T19:12:00.044+0000] {taskinstance.py:3157} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='process_exercises' AIRFLOW_CTX_TASK_ID='process_exercises' AIRFLOW_CTX_EXECUTION_DATE='2025-07-09T19:11:55.126986+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-07-09T19:11:55.126986+00:00'
[2025-07-09T19:12:00.045+0000] {taskinstance.py:740} INFO - ::endgroup::
[2025-07-09T19:12:00.081+0000] {base.py:84} INFO - Retrieving connection 'spark-ebsim'
[2025-07-09T19:12:00.090+0000] {spark_submit.py:474} INFO - Spark-Submit cmd: spark-submit --master spark://spark-master:7077 --jars /opt/bitnami/spark/jars/iceberg-spark-runtime-3.5_2.12-1.8.1.jar,/opt/bitnami/spark/jars/nessie-spark-extensions-3.5_2.12-0.103.2.jar,/opt/bitnami/spark/jars/nessie-client-0.103.2.jar,/opt/bitnami/spark/jars/hadoop-aws-3.3.2.jar,/opt/bitnami/spark/jars/aws-java-sdk-bundle-1.12.262.jar, --name arrow-spark --deploy-mode client /opt/bitnami/spark/jobs/process_exercises_job.py {"id": "CA-SUL-virtual-12345-2025-07-09", "manifest_entry": {"data": {"createdAt": "2025-07-09T19:11:22.499Z", "origin": "CA-SUL", "path": "exercises/CA-SUL-virtual-12345-2025-07-09"}, "id": "CA-SUL-virtual-12345-2025-07-09"}, "metadata": {"createdAt": "2025-07-09T19:11:22.499Z", "description": "12345", "documents": [{"description": "", "id": "1752088298151", "mimetype": "application/pdf", "name": "Op Jacui IV - Anexo A - Inteligencia.pdf", "path": "exercises/CA-SUL-virtual-12345-2025-07-09/documents/Op Jacui IV - Anexo A - Inteligencia.pdf", "size": 47523, "uploadedAt": "2025-07-09T19:11:38.151Z"}], "endDate": "2025-07-10", "id": "CA-SUL-virtual-12345-2025-07-09", "name": "Exerc\u00edcio Virtual", "origin": "CA-SUL", "simulationType": "virtual", "simulations": [{"id": "172.19.0.6_2025-07-09", "movedAt": "2025-07-09T19:11:46.029Z", "name": "172.19.0.6_2025-07-09", "originalPath": "staging/172.19.0.6_2025-07-09", "path": "exercises/CA-SUL-virtual-12345-2025-07-09/simulations/172.19.0.6_2025-07-09"}], "startDate": "2025-07-09", "trainedForce": "12345v", "updatedAt": "2025-07-09T19:11:46.029Z", "wikiUrl": ""}}
[2025-07-09T19:12:02.252+0000] {spark_submit.py:644} INFO - 25/07/09 19:12:02 WARN NativeCodeLoader: Unable to load native-hadoop library for your platform... using builtin-java classes where applicable
[2025-07-09T19:12:03.769+0000] {spark_submit.py:644} INFO - [PROCESS EXERCISES] ℹ️ Iniciando processamento otimizado de exercícios
[2025-07-09T19:12:03.770+0000] {spark_submit.py:644} INFO - [PROCESS EXERCISES] ℹ️ Exercício selecionado: CA-SUL-virtual-12345-2025-07-09
[2025-07-09T19:12:03.771+0000] {spark_submit.py:644} INFO - [PROCESS EXERCISES] 🔄 Criando sessão Spark otimizada
[2025-07-09T19:12:04.395+0000] {spark_submit.py:644} INFO - 25/07/09 19:12:04 INFO SparkContext: Running Spark version 3.5.5
[2025-07-09T19:12:04.396+0000] {spark_submit.py:644} INFO - 25/07/09 19:12:04 INFO SparkContext: OS info Linux, 5.15.167.4-microsoft-standard-WSL2, amd64
[2025-07-09T19:12:04.397+0000] {spark_submit.py:644} INFO - 25/07/09 19:12:04 INFO SparkContext: Java version 17.0.15
[2025-07-09T19:12:04.421+0000] {spark_submit.py:644} INFO - 25/07/09 19:12:04 INFO ResourceUtils: ==============================================================
[2025-07-09T19:12:04.422+0000] {spark_submit.py:644} INFO - 25/07/09 19:12:04 INFO ResourceUtils: No custom resources configured for spark.driver.
[2025-07-09T19:12:04.422+0000] {spark_submit.py:644} INFO - 25/07/09 19:12:04 INFO ResourceUtils: ==============================================================
[2025-07-09T19:12:04.423+0000] {spark_submit.py:644} INFO - 25/07/09 19:12:04 INFO SparkContext: Submitted application: ProcessExercisesOptimized
[2025-07-09T19:12:04.438+0000] {spark_submit.py:644} INFO - 25/07/09 19:12:04 INFO ResourceProfile: Default ResourceProfile created, executor resources: Map(memory -> name: memory, amount: 2048, script: , vendor: , offHeap -> name: offHeap, amount: 0, script: , vendor: ), task resources: Map(cpus -> name: cpus, amount: 1.0)
[2025-07-09T19:12:04.446+0000] {spark_submit.py:644} INFO - 25/07/09 19:12:04 INFO ResourceProfile: Limiting resource is cpu
[2025-07-09T19:12:04.446+0000] {spark_submit.py:644} INFO - 25/07/09 19:12:04 INFO ResourceProfileManager: Added ResourceProfile id: 0
[2025-07-09T19:12:04.486+0000] {spark_submit.py:644} INFO - 25/07/09 19:12:04 INFO SecurityManager: Changing view acls to: airflow
[2025-07-09T19:12:04.487+0000] {spark_submit.py:644} INFO - 25/07/09 19:12:04 INFO SecurityManager: Changing modify acls to: airflow
[2025-07-09T19:12:04.488+0000] {spark_submit.py:644} INFO - 25/07/09 19:12:04 INFO SecurityManager: Changing view acls groups to:
[2025-07-09T19:12:04.488+0000] {spark_submit.py:644} INFO - 25/07/09 19:12:04 INFO SecurityManager: Changing modify acls groups to:
[2025-07-09T19:12:04.489+0000] {spark_submit.py:644} INFO - 25/07/09 19:12:04 INFO SecurityManager: SecurityManager: authentication disabled; ui acls disabled; users with view permissions: airflow; groups with view permissions: EMPTY; users with modify permissions: airflow; groups with modify permissions: EMPTY
[2025-07-09T19:12:04.669+0000] {spark_submit.py:644} INFO - 25/07/09 19:12:04 INFO Utils: Successfully started service 'sparkDriver' on port 41203.
[2025-07-09T19:12:04.690+0000] {spark_submit.py:644} INFO - 25/07/09 19:12:04 INFO SparkEnv: Registering MapOutputTracker
[2025-07-09T19:12:04.717+0000] {spark_submit.py:644} INFO - 25/07/09 19:12:04 INFO SparkEnv: Registering BlockManagerMaster
[2025-07-09T19:12:04.729+0000] {spark_submit.py:644} INFO - 25/07/09 19:12:04 INFO BlockManagerMasterEndpoint: Using org.apache.spark.storage.DefaultTopologyMapper for getting topology information
[2025-07-09T19:12:04.734+0000] {spark_submit.py:644} INFO - 25/07/09 19:12:04 INFO BlockManagerMasterEndpoint: BlockManagerMasterEndpoint up
[2025-07-09T19:12:04.737+0000] {spark_submit.py:644} INFO - 25/07/09 19:12:04 INFO SparkEnv: Registering BlockManagerMasterHeartbeat
[2025-07-09T19:12:04.769+0000] {spark_submit.py:644} INFO - 25/07/09 19:12:04 INFO DiskBlockManager: Created local directory at /tmp/blockmgr-82e2bc1e-9e31-4c0b-95f8-8a2e72541a31
[2025-07-09T19:12:04.779+0000] {spark_submit.py:644} INFO - 25/07/09 19:12:04 INFO MemoryStore: MemoryStore started with capacity 434.4 MiB
[2025-07-09T19:12:04.793+0000] {spark_submit.py:644} INFO - 25/07/09 19:12:04 INFO SparkEnv: Registering OutputCommitCoordinator
[2025-07-09T19:12:04.911+0000] {spark_submit.py:644} INFO - 25/07/09 19:12:04 INFO JettyUtils: Start Jetty 0.0.0.0:4040 for SparkUI
[2025-07-09T19:12:04.967+0000] {spark_submit.py:644} INFO - 25/07/09 19:12:04 INFO Utils: Successfully started service 'SparkUI' on port 4040.
[2025-07-09T19:12:05.004+0000] {spark_submit.py:644} INFO - 25/07/09 19:12:05 INFO SparkContext: Added JAR file:///opt/bitnami/spark/jars/iceberg-spark-runtime-3.5_2.12-1.8.1.jar at spark://38b7408c68d7:41203/jars/iceberg-spark-runtime-3.5_2.12-1.8.1.jar with timestamp 1752088324389
[2025-07-09T19:12:05.005+0000] {spark_submit.py:644} INFO - 25/07/09 19:12:05 INFO SparkContext: Added JAR file:///opt/bitnami/spark/jars/nessie-spark-extensions-3.5_2.12-0.103.2.jar at spark://38b7408c68d7:41203/jars/nessie-spark-extensions-3.5_2.12-0.103.2.jar with timestamp 1752088324389
[2025-07-09T19:12:05.006+0000] {spark_submit.py:644} INFO - 25/07/09 19:12:05 INFO SparkContext: Added JAR file:///opt/bitnami/spark/jars/nessie-client-0.103.2.jar at spark://38b7408c68d7:41203/jars/nessie-client-0.103.2.jar with timestamp 1752088324389
[2025-07-09T19:12:05.007+0000] {spark_submit.py:644} INFO - 25/07/09 19:12:05 INFO SparkContext: Added JAR file:///opt/bitnami/spark/jars/hadoop-aws-3.3.2.jar at spark://38b7408c68d7:41203/jars/hadoop-aws-3.3.2.jar with timestamp 1752088324389
[2025-07-09T19:12:05.008+0000] {spark_submit.py:644} INFO - 25/07/09 19:12:05 INFO SparkContext: Added JAR file:///opt/bitnami/spark/jars/aws-java-sdk-bundle-1.12.262.jar at spark://38b7408c68d7:41203/jars/aws-java-sdk-bundle-1.12.262.jar with timestamp 1752088324389
[2025-07-09T19:12:05.095+0000] {spark_submit.py:644} INFO - 25/07/09 19:12:05 INFO StandaloneAppClient$ClientEndpoint: Connecting to master spark://spark-master:7077...
[2025-07-09T19:12:05.157+0000] {spark_submit.py:644} INFO - 25/07/09 19:12:05 INFO TransportClientFactory: Successfully created connection to spark-master/172.19.0.5:7077 after 43 ms (0 ms spent in bootstraps)
[2025-07-09T19:12:05.353+0000] {spark_submit.py:644} INFO - 25/07/09 19:12:05 INFO StandaloneSchedulerBackend: Connected to Spark cluster with app ID app-20250709191205-0014
[2025-07-09T19:12:05.360+0000] {spark_submit.py:644} INFO - 25/07/09 19:12:05 INFO Utils: Successfully started service 'org.apache.spark.network.netty.NettyBlockTransferService' on port 37163.
[2025-07-09T19:12:05.360+0000] {spark_submit.py:644} INFO - 25/07/09 19:12:05 INFO NettyBlockTransferService: Server created on 38b7408c68d7:37163
[2025-07-09T19:12:05.361+0000] {spark_submit.py:644} INFO - 25/07/09 19:12:05 INFO BlockManager: Using org.apache.spark.storage.RandomBlockReplicationPolicy for block replication policy
[2025-07-09T19:12:05.366+0000] {spark_submit.py:644} INFO - 25/07/09 19:12:05 INFO BlockManagerMaster: Registering BlockManager BlockManagerId(driver, 38b7408c68d7, 37163, None)
[2025-07-09T19:12:05.369+0000] {spark_submit.py:644} INFO - 25/07/09 19:12:05 INFO BlockManagerMasterEndpoint: Registering block manager 38b7408c68d7:37163 with 434.4 MiB RAM, BlockManagerId(driver, 38b7408c68d7, 37163, None)
[2025-07-09T19:12:05.370+0000] {spark_submit.py:644} INFO - 25/07/09 19:12:05 INFO BlockManagerMaster: Registered BlockManager BlockManagerId(driver, 38b7408c68d7, 37163, None)
[2025-07-09T19:12:05.371+0000] {spark_submit.py:644} INFO - 25/07/09 19:12:05 INFO BlockManager: Initialized BlockManager: BlockManagerId(driver, 38b7408c68d7, 37163, None)
[2025-07-09T19:12:05.384+0000] {spark_submit.py:644} INFO - 25/07/09 19:12:05 INFO StandaloneAppClient$ClientEndpoint: Executor added: app-20250709191205-0014/0 on worker-20250709171731-172.19.0.7-37405 (172.19.0.7:37405) with 2 core(s)
[2025-07-09T19:12:05.386+0000] {spark_submit.py:644} INFO - 25/07/09 19:12:05 INFO StandaloneSchedulerBackend: Granted executor ID app-20250709191205-0014/0 on hostPort 172.19.0.7:37405 with 2 core(s), 2.0 GiB RAM
[2025-07-09T19:12:05.467+0000] {spark_submit.py:644} INFO - 25/07/09 19:12:05 INFO StandaloneAppClient$ClientEndpoint: Executor updated: app-20250709191205-0014/0 is now RUNNING
[2025-07-09T19:12:05.649+0000] {spark_submit.py:644} INFO - 25/07/09 19:12:05 INFO StandaloneSchedulerBackend: SchedulerBackend is ready for scheduling beginning after reached minRegisteredResourcesRatio: 0.0
[2025-07-09T19:12:05.889+0000] {spark_submit.py:644} INFO - [PROCESS EXERCISES] ✅ Sessão Spark otimizada criada com sucesso
[2025-07-09T19:12:06.713+0000] {spark_submit.py:644} INFO - [PROCESS EXERCISES] ✅ Processador de exercícios inicializado com otimizações
[2025-07-09T19:12:06.714+0000] {spark_submit.py:644} INFO - [PROCESS EXERCISES] 🔄 Criando namespace nessie.exercises
[2025-07-09T19:12:09.803+0000] {spark_submit.py:644} INFO - [PROCESS EXERCISES] ✅ Namespace criado/verificado com sucesso
[2025-07-09T19:12:09.803+0000] {spark_submit.py:644} INFO - [PROCESS EXERCISES] 🔄 === FASE 1: PROCESSAMENTO DE DOCUMENTOS ===
[2025-07-09T19:12:09.804+0000] {spark_submit.py:644} INFO - [PROCESS EXERCISES] 🔄 Iniciando processamento de documentos PDF
[2025-07-09T19:12:09.805+0000] {spark_submit.py:644} INFO - [PROCESS EXERCISES] ℹ️ Processando 1 arquivos PDF
[2025-07-09T19:12:09.805+0000] {spark_submit.py:644} INFO - [PROCESS EXERCISES] ℹ️ Processando lote 1: 1 arquivos
[2025-07-09T19:12:10.042+0000] {spark_submit.py:644} INFO - 25/07/09 19:12:10 WARN MetricsConfig: Cannot locate configuration: tried hadoop-metrics2-s3a-file-system.properties,hadoop-metrics2.properties
[2025-07-09T19:12:25.402+0000] {spark_submit.py:644} INFO - [PROCESS EXERCISES] ✅ Lote processado: 1 registros
[2025-07-09T19:12:28.855+0000] {spark_submit.py:644} INFO - [PROCESS EXERCISES] ✅ Total de PDFs processados: 1 registros
[2025-07-09T19:12:32.068+0000] {spark_submit.py:644} INFO - [PROCESS EXERCISES] ℹ️ Salvando 1 registros na tabela nessie.exercises.pdf_documents
[2025-07-09T19:12:33.207+0000] {spark_submit.py:644} INFO - [PROCESS EXERCISES] 🔄 Salvando tabela nessie.exercises.pdf_documents em modo append (1 partições)
[2025-07-09T19:12:36.814+0000] {spark_submit.py:644} INFO - [PROCESS EXERCISES] ✅ Tabela nessie.exercises.pdf_documents salva com sucesso (1 registros)
[2025-07-09T19:12:36.815+0000] {spark_submit.py:644} INFO - [PROCESS EXERCISES] 🔄 === FASE 2: PROCESSAMENTO DE SIMULAÇÕES ===
[2025-07-09T19:12:36.816+0000] {spark_submit.py:644} INFO - [PROCESS EXERCISES] 🔄 Iniciando processamento de simulações
[2025-07-09T19:12:36.816+0000] {spark_submit.py:644} INFO - [PROCESS EXERCISES] ℹ️ Buscando arquivos em: s3a://bronze/exercises/CA-SUL-virtual-12345-2025-07-09/simulations/172.19.0.6_2025-07-09/
[2025-07-09T19:12:37.409+0000] {spark_submit.py:644} INFO - [PROCESS EXERCISES] ℹ️ Encontrados 1 arquivos DIS (.bin)
[2025-07-09T19:12:37.410+0000] {spark_submit.py:644} INFO - [PROCESS EXERCISES] ℹ️ Processando 1 arquivos DIS
[2025-07-09T19:12:40.079+0000] {spark_submit.py:644} INFO - [PROCESS EXERCISES] ✅ Arquivos DIS processados: 1 arquivos (400 registros)
[2025-07-09T19:12:40.079+0000] {spark_submit.py:644} INFO - [PROCESS EXERCISES] ℹ️ Tipo de mensagem de simulação: DIS
[2025-07-09T19:12:40.771+0000] {spark_submit.py:644} INFO - [PROCESS EXERCISES] ℹ️ Salvando 400 registros na tabela nessie.exercises.dis_messages
[2025-07-09T19:12:40.953+0000] {spark_submit.py:644} INFO - [PROCESS EXERCISES] 🔄 Salvando tabela nessie.exercises.dis_messages em modo append (1 partições)
[2025-07-09T19:12:41.108+0000] {spark_submit.py:644} INFO - [PROCESS EXERCISES] ❌ Erro ao salvar tabela nessie.exercises.dis_messages: [INSERT_COLUMN_ARITY_MISMATCH.TOO_MANY_DATA_COLUMNS] Cannot write to `nessie`.`exercises`.`dis_messages`, the reason is too many data columns:
[2025-07-09T19:12:41.109+0000] {spark_submit.py:644} INFO - Table columns: `exercise_id`, `entity_id`, `force_id`, `kind`, `latitude`, `longitude`, `altitude`, `timestamp`, `processed_at`.
[2025-07-09T19:12:41.109+0000] {spark_submit.py:644} INFO - Data columns: `exercise_id`, `source_file`, `entity_id`, `force_id`, `kind`, `latitude`, `longitude`, `altitude`, `timestamp`, `processed_at`.
[2025-07-09T19:12:41.112+0000] {spark_submit.py:644} INFO - [PROCESS EXERCISES] ❌ Erro crítico durante o processamento: Falha ao salvar tabela nessie.exercises.dis_messages: [INSERT_COLUMN_ARITY_MISMATCH.TOO_MANY_DATA_COLUMNS] Cannot write to `nessie`.`exercises`.`dis_messages`, the reason is too many data columns:
[2025-07-09T19:12:41.112+0000] {spark_submit.py:644} INFO - Table columns: `exercise_id`, `entity_id`, `force_id`, `kind`, `latitude`, `longitude`, `altitude`, `timestamp`, `processed_at`.
[2025-07-09T19:12:41.113+0000] {spark_submit.py:644} INFO - Data columns: `exercise_id`, `source_file`, `entity_id`, `force_id`, `kind`, `latitude`, `longitude`, `altitude`, `timestamp`, `processed_at`.
[2025-07-09T19:12:41.113+0000] {spark_submit.py:644} INFO - [PROCESS EXERCISES] 🔄 Finalizando sessão Spark
[2025-07-09T19:12:41.694+0000] {spark_submit.py:644} INFO - [PROCESS EXERCISES] ✅ Sessão Spark finalizada com sucesso
[2025-07-09T19:12:42.387+0000] {taskinstance.py:3336} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 776, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 742, in _execute_callable
    return ExecutionCallableRunner(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 424, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/apache/spark/operators/spark_submit.py", line 197, in execute
    self._hook.submit(self.application)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/apache/spark/hooks/spark_submit.py", line 566, in submit
    raise AirflowException(
airflow.exceptions.AirflowException: Cannot execute: spark-submit --master spark://spark-master:7077 --jars /opt/bitnami/spark/jars/iceberg-spark-runtime-3.5_2.12-1.8.1.jar,/opt/bitnami/spark/jars/nessie-spark-extensions-3.5_2.12-0.103.2.jar,/opt/bitnami/spark/jars/nessie-client-0.103.2.jar,/opt/bitnami/spark/jars/hadoop-aws-3.3.2.jar,/opt/bitnami/spark/jars/aws-java-sdk-bundle-1.12.262.jar, --name arrow-spark --deploy-mode client /opt/bitnami/spark/jobs/process_exercises_job.py {"id": "CA-SUL-virtual-12345-2025-07-09", "manifest_entry": {"data": {"createdAt": "2025-07-09T19:11:22.499Z", "origin": "CA-SUL", "path": "exercises/CA-SUL-virtual-12345-2025-07-09"}, "id": "CA-SUL-virtual-12345-2025-07-09"}, "metadata": {"createdAt": "2025-07-09T19:11:22.499Z", "description": "12345", "documents": [{"description": "", "id": "1752088298151", "mimetype": "application/pdf", "name": "Op Jacui IV - Anexo A - Inteligencia.pdf", "path": "exercises/CA-SUL-virtual-12345-2025-07-09/documents/Op Jacui IV - Anexo A - Inteligencia.pdf", "size": 47523, "uploadedAt": "2025-07-09T19:11:38.151Z"}], "endDate": "2025-07-10", "id": "CA-SUL-virtual-12345-2025-07-09", "name": "Exerc\u00edcio Virtual", "origin": "CA-SUL", "simulationType": "virtual", "simulations": [{"id": "172.19.0.6_2025-07-09", "movedAt": "2025-07-09T19:11:46.029Z", "name": "172.19.0.6_2025-07-09", "originalPath": "staging/172.19.0.6_2025-07-09", "path": "exercises/CA-SUL-virtual-12345-2025-07-09/simulations/172.19.0.6_2025-07-09"}], "startDate": "2025-07-09", "trainedForce": "12345v", "updatedAt": "2025-07-09T19:11:46.029Z", "wikiUrl": ""}}. Error code is: 1.
[2025-07-09T19:12:42.580+0000] {taskinstance.py:1242} INFO - Marking task as FAILED. dag_id=process_exercises, task_id=process_exercises, run_id=manual__2025-07-09T19:11:55.126986+00:00, execution_date=20250709T191155, start_date=20250709T191159, end_date=20250709T191242
[2025-07-09T19:12:42.809+0000] {taskinstance.py:349} INFO - ::group::Post task execution logs
[2025-07-09T19:12:42.810+0000] {standard_task_runner.py:124} ERROR - Failed to execute job 250 for task process_exercises (Cannot execute: spark-submit --master spark://spark-master:7077 --jars /opt/bitnami/spark/jars/iceberg-spark-runtime-3.5_2.12-1.8.1.jar,/opt/bitnami/spark/jars/nessie-spark-extensions-3.5_2.12-0.103.2.jar,/opt/bitnami/spark/jars/nessie-client-0.103.2.jar,/opt/bitnami/spark/jars/hadoop-aws-3.3.2.jar,/opt/bitnami/spark/jars/aws-java-sdk-bundle-1.12.262.jar, --name arrow-spark --deploy-mode client /opt/bitnami/spark/jobs/process_exercises_job.py {"id": "CA-SUL-virtual-12345-2025-07-09", "manifest_entry": {"data": {"createdAt": "2025-07-09T19:11:22.499Z", "origin": "CA-SUL", "path": "exercises/CA-SUL-virtual-12345-2025-07-09"}, "id": "CA-SUL-virtual-12345-2025-07-09"}, "metadata": {"createdAt": "2025-07-09T19:11:22.499Z", "description": "12345", "documents": [{"description": "", "id": "1752088298151", "mimetype": "application/pdf", "name": "Op Jacui IV - Anexo A - Inteligencia.pdf", "path": "exercises/CA-SUL-virtual-12345-2025-07-09/documents/Op Jacui IV - Anexo A - Inteligencia.pdf", "size": 47523, "uploadedAt": "2025-07-09T19:11:38.151Z"}], "endDate": "2025-07-10", "id": "CA-SUL-virtual-12345-2025-07-09", "name": "Exerc\u00edcio Virtual", "origin": "CA-SUL", "simulationType": "virtual", "simulations": [{"id": "172.19.0.6_2025-07-09", "movedAt": "2025-07-09T19:11:46.029Z", "name": "172.19.0.6_2025-07-09", "originalPath": "staging/172.19.0.6_2025-07-09", "path": "exercises/CA-SUL-virtual-12345-2025-07-09/simulations/172.19.0.6_2025-07-09"}], "startDate": "2025-07-09", "trainedForce": "12345v", "updatedAt": "2025-07-09T19:11:46.029Z", "wikiUrl": ""}}. Error code is: 1.; 5347)
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py", line 117, in _start_by_fork
    ret = args.func(args, dag=self.dag)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/cli_config.py", line 49, in command
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/cli.py", line 116, in wrapper
    return f(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 483, in task_run
    task_return_code = _run_task_by_selected_method(args, _dag, ti)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 256, in _run_task_by_selected_method
    return _run_raw_task(args, ti)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 341, in _run_raw_task
    return ti._run_raw_task(
           ^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/session.py", line 97, in wrapper
    return func(*args, session=session, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3029, in _run_raw_task
    return _run_raw_task(
           ^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 282, in _run_raw_task
    TaskInstance._execute_task_with_callbacks(
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3184, in _execute_task_with_callbacks
    result = self._execute_task(context, task_orig)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3208, in _execute_task
    return _execute_task(self, context, task_orig)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 776, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 742, in _execute_callable
    return ExecutionCallableRunner(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 424, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/apache/spark/operators/spark_submit.py", line 197, in execute
    self._hook.submit(self.application)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/apache/spark/hooks/spark_submit.py", line 566, in submit
    raise AirflowException(
airflow.exceptions.AirflowException: Cannot execute: spark-submit --master spark://spark-master:7077 --jars /opt/bitnami/spark/jars/iceberg-spark-runtime-3.5_2.12-1.8.1.jar,/opt/bitnami/spark/jars/nessie-spark-extensions-3.5_2.12-0.103.2.jar,/opt/bitnami/spark/jars/nessie-client-0.103.2.jar,/opt/bitnami/spark/jars/hadoop-aws-3.3.2.jar,/opt/bitnami/spark/jars/aws-java-sdk-bundle-1.12.262.jar, --name arrow-spark --deploy-mode client /opt/bitnami/spark/jobs/process_exercises_job.py {"id": "CA-SUL-virtual-12345-2025-07-09", "manifest_entry": {"data": {"createdAt": "2025-07-09T19:11:22.499Z", "origin": "CA-SUL", "path": "exercises/CA-SUL-virtual-12345-2025-07-09"}, "id": "CA-SUL-virtual-12345-2025-07-09"}, "metadata": {"createdAt": "2025-07-09T19:11:22.499Z", "description": "12345", "documents": [{"description": "", "id": "1752088298151", "mimetype": "application/pdf", "name": "Op Jacui IV - Anexo A - Inteligencia.pdf", "path": "exercises/CA-SUL-virtual-12345-2025-07-09/documents/Op Jacui IV - Anexo A - Inteligencia.pdf", "size": 47523, "uploadedAt": "2025-07-09T19:11:38.151Z"}], "endDate": "2025-07-10", "id": "CA-SUL-virtual-12345-2025-07-09", "name": "Exerc\u00edcio Virtual", "origin": "CA-SUL", "simulationType": "virtual", "simulations": [{"id": "172.19.0.6_2025-07-09", "movedAt": "2025-07-09T19:11:46.029Z", "name": "172.19.0.6_2025-07-09", "originalPath": "staging/172.19.0.6_2025-07-09", "path": "exercises/CA-SUL-virtual-12345-2025-07-09/simulations/172.19.0.6_2025-07-09"}], "startDate": "2025-07-09", "trainedForce": "12345v", "updatedAt": "2025-07-09T19:11:46.029Z", "wikiUrl": ""}}. Error code is: 1.
[2025-07-09T19:12:42.903+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 1
[2025-07-09T19:12:42.934+0000] {taskinstance.py:3924} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-07-09T19:12:43.143+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
